daemon/graphdriver/windows/windows.go

```diff
@@ -244,7 +244,18 @@ func (d *Driver) Remove(id string) error {
 		return err
 	}
 	os.RemoveAll(filepath.Join(d.info.HomeDir, "sysfile-backups", rID)) // ok to fail
```

Comment:
I believe you can remove this line now. I don't see where we ever populate this directory.